movo_common/movo_description/urdf/movo_components/movo_pan_tilt.urdf.xacro

```diff
@@ -26,8 +26,17 @@ LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
 NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS 
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 -->
+
 <robot name="movo_pan_tilt" xmlns:xacro="http://ros.org/wiki/xacro">
-    <xacro:include filename="$(find movo_description)/urdf/sensors/_d435.urdf.xacro"/>
+    <!-- swap out these d435 realsense files for sim (_d435_sim) and for real.  use "export MOVO_SIM=true" to set sim as environment variable-->
```

Comment:
would it be better if this took in the arg indicating if this is sim or not (the arg already exists, its essential for us to not launch nodes that interface with HW in sim)

Reply:
Yeah a command line variable does sound better. Where is the argument located? I can try to pass it through to the file

Reply:
ok I think I got it. I initialize a "sim" arg that is only ever set to "true" in the simulation launch file and is false everywhere else. When running sim, the parameter is passed through down to the xacro and should always use the HW unless the movo_sim/movo.launch file is called when running sim